src/app/game/flip-clock/page.tsx

```
@@ -0,0 +1,309 @@
+              onChange={(e) => setCustomTime(prev => ({ ...prev, seconds: parseInt(e.target.value) || 0 }))}
+              disabled={isCountdown}
+            />
+          </div>
+        </div>
+
+        <div className={styles.buttons}>
+          <button 
+            onClick={startCountdown}
+            disabled={isCountdown}
+            className={styles.startButton}
+          >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)